apps/docs/pages/guides/getting-started/quickstarts/openai.mdx

```diff
@@ -0,0 +1,163 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+import StepHikeCompact from '~/components/StepHikeCompact'
+
+export const meta = {
+  title: 'Supabase with OpenAI',
+  subtitle: 'Using Supabase with OpenAI.',
+}
+
+AI products are transforming the world today with their amazing capabilities and benefits. AI products can help us solve complex problems, automate tedious tasks, enhance our creativity and productivity, and make our lives easier and more enjoyable. Some cool example of it is Supabase Clippy released not so long ago.
+
+But how can you create your own AI products without spending too much time and money? How can you leverage the power of AI to build innovative solutions for your business or personal needs? That's where Supabase comes in. It offers an easy and efficient way to manage your data for AI. Supabase is an open source platform that provides a variety of tools for data management, including data storage, authentication, and APIs.
+
+If you want to learn more about how to use Supabase for AI products, you're in luck. We have prepared a comprehensive guide that will teach you everything you need to know. You'll learn how to set up your Supabase project, use OpenAI to do natural language processing, text summarization, and more to make your own product for search and question answering.
```

Comment:
This whole section up to here is a bit too much like advertising. Can we reframe it more for the Docs - just informative and succinct?

Reply:
I've removed this text and just left enough info in the subtitle instead as this is what we are doing for the other quickstart guides.